src/app/content/components/Assigned.tsx

```diff
@@ -90,7 +90,7 @@ export default () => {
     <ErrorModal />
     <ErrorBoundary>
       <AssignedTopBar section={section} />
-      <Page>
+      <Page inAssigned={true} >
```

Comment:
i would pass props for what we actually want to do here, rather than leaking context and forcing more logic into the page component. maybe `<Page lockNavigation>` to indicate that we don't want to allow navigating to different pages in this view.